### ISSUE TITLE
The role of miRNAs in colorectal cancer progression and chemoradiotherapy

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#30]
## TL;DR

MicroRNAs (miRNAs) are a class of small noncoding RNAs that can affect a variety of cellular and molecular targets. Depending on the cell environment in which the information is expressed, miRNAs can serve as a CRC suppressor or promoter and play essential roles in several biological processes. In this review, we summarized the relationship between miRNAs and proliferation, metastasis, angiogenesis, autophagy, apoptosis, and the chemoradiotherapy of CRC, revealing that relevant miRNAs could serve as potential targets for CRC therapy.

### Paper Link

https://www.sciencedirect.com/science/article/pii/S0753332220312920

### Author/Institution

 Ning Zhang (Anhui Medical University)

## Overview

miRNAs play a systematic role in CRC proliferation, metastasis, angiogenesis, autophagy, apoptosis, and chemoradiotherapy, regulating multiple related signing pathways such as the Wnt/β-catenin, PI3K/AKT/mTOR and ERK-MAPK pathway. A large number of miRNAs are regarded as oncogenes or tumor suppressors; therefore, understanding the mechanism of miRNAs in tumorigenesis serves an important role in cancer therapy. Although miRNA has great potential in the [treatment](https://www.sciencedirect.com/topics/medicine-and-dentistry/therapeutic-procedure) of CRC, the practical use of miRNA in clinical practice remains to be investigated. How do we identify miRNAs that are involved in the treatment response? How do we avoid the side effects of miRNA therapy? The role and mechanism of miRNAs in CRC still requires in-depth research.

## Cite

Ning Zhang, Xianyu Hu, Yinan Du, Juan Du,
The role of miRNAs in colorectal cancer progression and chemoradiotherapy,
Biomedicine & Pharmacotherapy,
Volume 134,
2021,
111099,
ISSN 0753-3322,
https://doi.org/10.1016/j.biopha.2020.111099.
(https://www.sciencedirect.com/science/article/pii/S0753332220312920)
